charts/hono/templates/_helpers.tpl

```diff
@@ -350,6 +366,18 @@ Optionally, the scope my contain key
   readOnly: true
 {{- end }}
 {{- end }}
+{{- if .dot }}
+{{- if ( contains "quarkus" .dot.Values.honoImagesType ) }}
+- name: "config"
+{{- if ( eq "quarkus" .dot.Values.honoImagesType ) }}
+  mountPath: "/deployments/config"
+{{- end }}
+{{- if ( eq "quarkus-native" .dot.Values.honoImagesType ) }}
+  mountPath: "/work/config"
+{{- end }}
+  readOnly: true
+{{- end }}
+{{- end }}
```

Comment:
my understanding is that Quarkus expects to find the config files in different locations than `/etc/hono` and also cannot be configured to read them from `/etc/hono` (which I would find very strange), right?
If so, then I think that the solution should not be to duplicate the config volume but instead simply mount the config volume to the Quarkus specific path(s). This would simply require to add the if block as an *else if* block to the original volume mount, wouldn't it?

Reply:
The issue is only location of the `application.yaml` file. There are some more files in the secret that are referenced from the app config, so I tried to keep changes minimal. As you said, I see the issue of not having a configurable app config file as an issue we need to deal in Quarkus. In that sense I would not invest too much resources in this as it's only a temporary state.